src/javaApi.cpp

```diff
@@ -37,6 +37,7 @@ Java_one_profiler_AsyncProfiler_start0(JNIEnv* env, jobject unused, jstring even
         args._event = event_str;
         args._interval = interval;
     }
+    Log::open(args._log, args._loglevel);
```

Comment:
Was this added intentionally? `start0` does not accept arguments other than `event` and `interval`, so `_log` and `_loglevel` are always empty here.

Reply:
I've removed this one.